internal/ethapi/transaction_args.go

```diff
@@ -171,6 +171,10 @@ func (args *TransactionArgs) setFeeDefaults(ctx context.Context, b Backend) erro
 
 // setLondonFeeDefaults fills in reasonable default fee values for unspecified fields.
 func (args *TransactionArgs) setLondonFeeDefaults(ctx context.Context, head *types.Header, b Backend) error {
+	// After London was activated, a gas price of zero is forbidden.
```

Comment:
It's not "forbidden" per se, there is a minimum basefee, so any gasPrice below a certain limit can never be executed. I think the limit is `7` or `8` or something like that